libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -2768,6 +2771,34 @@ void GCS_MAVLINK::send_vfr_hud()
         vfr_hud_climbrate());
 }
 
+#if AP_HYGROMETER_ENABLED
+void GCS_MAVLINK::send_hygrometer(const class AP_Hygrometer_Backend *sensor, const uint8_t instance) const
```

Comment:
```suggestion
void GCS_MAVLINK::send_hygrometer(const class AP_Hygrometer_Backend &sensor, const uint8_t instance) const
```

make the caller do the dereference right after making the nullptr check